### ISSUE TITLE
Allow dual function modifier to be right shift

### ISSUE THREAD

**Issue by senft** [Dygmalab/Bazecor#788]
It would be nice to be able to bind a dual function modifier to right shift. It is already possible to bind dual function modifiers to alt as well as alt gr/right alt. So why not shift and right shift?

**Comment by dobernoeder** [comment]
I agree, that would be very helpful. I use Linux with a function to enable/disable caps lock when I press right and left shift key on the same time. With my keyboard that is currently not possible to switch to caps lock due to that design.

**Comment by dobernoeder** [comment]
Looks like the newest Beta for .1.5.0 does have a modification in the new Layout Editor, Dygma presented in a YouTube Video this week.
![Bildschirmfoto vom 2024-09-07 01-18-49](https://github.com/user-attachments/assets/5bb943e5-e120-420e-8274-e567e91760f6)

Unfortunately, the Option to "Add a key on tap" is just working for the left shift to apply a key on simple tap. It is grayed out, when using right shift. It makes me optimistic that it might be added in the future :smile:

**Comment by MiquelDygma** [comment]
Hi!

First of all, my apologies for not replying to this thread sooner. 

This is something that's possible to do with superkeys, but they don't behave as nicely as dual-function keys (now, "Add a key on tap"). 

Our goal is to make superkeys behave as nicely 😊
